server/util/monitoring/monitoring.go

```diff
@@ -24,6 +25,9 @@ func RegisterMonitoringHandlers(mux *http.ServeMux) {
 
 	// Statusz page
 	mux.Handle("/statusz", statusz.Server())
+
+	// Flagz page
+	mux.HandleFunc("/flagz/", flagz.ServeHTTP)
```

Comment:
nit: remove trailing slash, flagz is the page not a route to another place (similar to /statusz)